### PR TITLE
1756-Added downtime incident management document

### LINKED ISSUE #1756: Create an incident response playbook
### Issue description

An incident response playbook is needed to ensure that the team can handle an application outage successfully. The playbook should include the actions that should be followed in case of incident. This includes but is not limited to:
- expectations for communication
- procedure for informing users

### Acceptance criteria

- [ ] Develop an incident response playbook
- [ ] Product team approves the corresponding PR

### Additional context

_No response_

### Links to other issues

related to: #1739 #1737 #[226 ](https://github.com/cisagov/get.gov/issues/226)

### PR DESCRIPTION
**Pull request by abroddrick**
## Ticket

Resolves #1756 

## Changes

- adds document for managing when the site is down
- links to a document for phone numbers

#### Satisfied acceptance criteria and met development standards

- [x] Met the acceptance criteria, or will meet them in a subsequent PR
- [ ] Created/modified automated tests
- [x] Added at least 2 developers as PR reviewers (only 1 will need to approve)
- [x] Messaged on Slack or in standup to notify the team that a PR is ready for review
- [ ] Changes to “how we do things” are documented in READMEs and or onboarding guide
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the associated migrations file has been commited.

#### Reviewed, tested, and left feedback about the changes

- [ ] Pulled this branch locally and tested it
- [ ] Reviewed this code and left comments
- [ ] Checked that all code is adequately covered by tests
- [ ] Made it clear which comments need to be addressed before this work is merged
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the associated migrations file has been commited.

#### Ensured code standards are met (Code reviewer)

- [ ] All new functions and methods are commented using plain language
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values
- [ ] (Rarely needed) Did dependency updates in Pipfile also get changed in requirements.txt?

### CODE REVIEW

File: docs/operations/runbooks/downtime_incident_management.md
Comment (PaulKuykendall) [inline docs/operations/runbooks/downtime_incident_management.md:9]: Should this be the new @dotgov-redalert slack channel?

File: docs/operations/runbooks/downtime_incident_management.md
Comment (PaulKuykendall) [inline docs/operations/runbooks/downtime_incident_management.md:27]: We can put some effort into an "incident" template to be used during the outage, to capture the details as they happen.

Comment (PaulKuykendall) [reply docs/operations/runbooks/downtime_incident_management.md:27]: Perhaps we pin the template to the redalert channel, so the reporter can grab it and start logging information.

File: docs/operations/runbooks/downtime_incident_management.md
Comment (PaulKuykendall) [inline docs/operations/runbooks/downtime_incident_management.md:3]: regarding the "critical bug in our code," we won't know the cause at the outset, only that the site is down. Also, it needs to be clear how this is different from a security incident.

**Review by PaulKuykendall — CHANGES_REQUESTED** [review]
Just a couple comments for you, that I think could be clarified.

File: docs/operations/runbooks/downtime_incident_management.md
Comment (abroddrick) [reply docs/operations/runbooks/downtime_incident_management.md:9]: No not everyone is in dotgov-redalert cahnnel. This is to make sure everyone is in the loop

File: docs/operations/runbooks/downtime_incident_management.md
Comment (abroddrick) [reply docs/operations/runbooks/downtime_incident_management.md:27]: There is a template already in progress

File: docs/operations/runbooks/downtime_incident_management.md
Comment (abroddrick) [reply docs/operations/runbooks/downtime_incident_management.md:3]: Actually the point here is when there is only something critical in our code, but I could add in or third parties. The point being I don't want to make this doc about defining what a security attack could be, etc. This is about things in our control directly.

Comment (abroddrick) [reply docs/operations/runbooks/downtime_incident_management.md]: language updated see what you think now

**Comment by abroddrick** [comment]
@PaulKuykendall, I made updates, re-review when you have a spare moment please

File: docs/operations/runbooks/downtime_incident_management.md
Comment (h-m-f-t) [inline docs/operations/runbooks/downtime_incident_management.md:25]: Add an @here to #dotgov-announce that the site is back up? Or reply to the prior @

File: docs/operations/runbooks/downtime_incident_management.md
Comment (h-m-f-t) [inline docs/operations/runbooks/downtime_incident_management.md:19]:
```suggestion
- If the problem is not solved within three hours, change the rules on the [Cloudflare's DNS dashboard](https://dash.cloudflare.com/) so that navigating to manage.get.gov redirects users to get.gov. This will help them see the banner on get.gov informing them that this is a known problem
```

Comment (h-m-f-t) [reply docs/operations/runbooks/downtime_incident_management.md:19]: This should probably link to an internal doc that specifies who has access to make these changes...

File: docs/operations/runbooks/downtime_incident_management.md
Comment (h-m-f-t) [inline docs/operations/runbooks/downtime_incident_management.md:17]: I don't love leaving this in comments. I think I'd prefer we had the text in a document with clear instructions and prepared/templated language.

**Comment by h-m-f-t** [comment]
We should also have as a post- todo to review these steps as a team and update as useful.

File: docs/operations/runbooks/downtime_incident_management.md
Comment (rachidatecs) [inline docs/operations/runbooks/downtime_incident_management.md:9]: fullstops after every bullet point.

Comment (abroddrick) [reply docs/operations/runbooks/downtime_incident_management.md:9]: what? Chaotically switching between having periods and not having periods doesn't make sense? 

Will change this

File: docs/operations/runbooks/downtime_incident_management.md
Comment (abroddrick) [reply docs/operations/runbooks/downtime_incident_management.md:17]: We could do that but then we would need the whole banner in this separate doc (which isn't a huge issue).
@Katherine-Osos I believe you are making the text for the banner anyways, are you already using a content word document for it? Perhaps that could be used for this purpose as well? If not we could just copy your finished language and the banner and place it in a separate doc for anyone to use in an emergency.

Comment (h-m-f-t) [reply docs/operations/runbooks/downtime_incident_management.md]: We could also keep the code and content for the banner in this repo somewhere.

File: docs/operations/runbooks/downtime_incident_management.md
Comment (h-m-f-t) [inline docs/operations/runbooks/downtime_incident_management.md:19]:
```suggestion
- If the problem is not solved within three hours, change the rules on the [Cloudflare's DNS dashboard](https://dash.cloudflare.com/) so that navigating to manage.get.gov redirects users to get.gov. (See [this document](https://docs.google.com/document/d/1PiWXpjBzbiKsSYqEo9Rkl72HMytMp7zTte9CI-vvwYw/edit) for how to make these changes and who can make them.) This will help them see the banner on get.gov informing them that this is a known problem.
```

Comment (h-m-f-t) [reply docs/operations/runbooks/downtime_incident_management.md:19]: @abroddrick I propose we test this redirect as configured after-hours sometime before this PR is merged.

**Review by h-m-f-t — CHANGES_REQUESTED** [review]
One small update and a request for a test.

File: docs/operations/runbooks/downtime_incident_management.md
Comment (abroddrick) [reply docs/operations/runbooks/downtime_incident_management.md:19]: We have a ticket to investigate if the redirect is the right call and how to best do the redirect. Do we then want to block this ticket by that one? Also do we want to test it on our actual site or with something like dotgov.gov? If we do our site, I think we should show a maintenance banner in advance saying we will be offline for maintenance between X and Y and maybe show a maintence banner on get.gov too. Thoughts?

Comment (h-m-f-t) [reply docs/operations/runbooks/downtime_incident_management.md:19]: Ah, I [see that now](https://github.com/cisagov/manage.get.gov/issues/1785) and thought we'd already done that given the language about redirects. 

I defer to you. We could drop this language and wait for research to inform our approach (and merge this now) or we could hold this for research and update it more then.

Comment (abroddrick) [reply docs/operations/runbooks/downtime_incident_management.md:19]: Good call, I just left this here and figured it would be updated in that ticket. However, yes I will remove that piece shortly and then we can just add it back in after we confirm that's the right step and have a document with the step by step instructions

Comment (h-m-f-t) [reply docs/operations/runbooks/downtime_incident_management.md:19]: 👍

**Review by h-m-f-t — APPROVED** [review]
🎉

**Comment by h-m-f-t** [comment]
Once merged, let's link to this document in [redalert's canvas](https://dhscisa.enterprise.slack.com/canvas/C06JDTB57J7).

**Pull request merged**
